app/assets/stylesheets/layout/_authorslist.scss

```diff
@@ -0,0 +1,50 @@
+.authorslist {
```

Comment:
Thoughts on separating the words with a hyphen or underscore for easier reading?